pkg/syncer/syncer.go

```diff
@@ -210,6 +210,14 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 
 	go specSyncer.Start(ctx, numSyncerThreads)
 	go statusSyncer.Start(ctx, numSyncerThreads)
+	// connect to create the reverse tunnels
+	go wait.UntilWithContext(ctx, func(ctx context.Context) {
+		klog.V(5).Infof("Starting tunnel for clusterName %s from pcluster", cfg.KCPClusterName, cfg.SyncTargetName)
+		err := startTunneler(ctx, upstreamConfig, downstreamConfig, cfg.KCPClusterName.String(), cfg.SyncTargetName)
```

Comment:
is this blocking? Why 1 min delay below?

Reply:
it's blocking, the 1 mins is to wait for reconnecting